### PR TITLE
CSP-1597 Provider Web content updates

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

**Review by shaun-downey-education — DISMISSED** [review]

File: src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/ShutterPages/PayeAornMatchedEmailNotLinkedHasRelationshipShutterPage.cshtml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <div class="govuk-grid-column-two-thirds">
         <p class="govuk-body">
             <div class="govuk-inset-text">
-                @Model.EmployerName <br /><br />
+                <p class="govuk-!-font-size-24">@Model.EmployerName </p>
```
Comment (PeterDavidWilliams) [inline src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/ShutterPages/PayeAornMatchedEmailNotLinkedHasRelationshipShutterPage.cshtml:19]: I believe wherever employer name appears it needs to be in all caps. The example given in the ticket is in all caps but I don't see here or in the model where it's set to all caps. Worth double checking with Olivia if it does need to be all caps

**Review by PeterDavidWilliams — COMMENTED** [review]
One thing to look at - an instance of employer name not being capitalised

File: src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/AddEmployerConfirmation.cshtml [hunk 2]
```diff
@@ -21,17 +21,17 @@
         </div>
         <div class="govuk-grid-column-two-thirds">
             <p class="govuk-body">
-                We’ve sent an email to @Model.LegalName.ToUpper() asking them to add you as a training provider with the permissions you requested. The request will expire in 14 days.
+                We’ve emailed @Model.LegalName.ToUpper() asking them to add you as a training provider with the permissions you requested. The request will expire in 14 days.
             </p>
             <p class="govuk-body">
-            If @Model.Email is not authorised to make changes to the account, we’ll also email the person at  @Model.LegalName.ToUpper() authorised to do this.
+            If <bold>@Model.Email</bold> is not authorised to make changes to the account, we’ll also email the person at  @Model.LegalName.ToUpper() authorised to do this.
```
[the remark this answers is not in the document]
Comment (MarkFCain) [reply src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/AddEmployerConfirmation.cshtml]: fixed

File: src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/ShutterPages/PayeAornMatchedEmailNotLinkedHasRelationshipShutterPage.cshtml (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/ShutterPages/PayeAornMatchedEmailNotLinkedHasRelationshipShutterPage.cshtml:19]: in this case, the viewmodel casts it to upper

**Review by PeterDavidWilliams — APPROVED** [review]

File: src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/ShutterPages/PayeAornMatchedEmailNotLinkedHasRelationshipShutterPage.cshtml (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/ShutterPages/PayeAornMatchedEmailNotLinkedHasRelationshipShutterPage.cshtml:19]: here: https://github.com/SkillsFundingAgency/das-provider-pr-web/blob/2323c03d8629d8f597fdc29b3ba128e85133e5e5/src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/SearchByPayeController.cs#L249

**Review by shaun-downey-education — DISMISSED** [review]

**Review by PeterDavidWilliams — APPROVED** [review]

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**